src/index.ts

```diff
@@ -1,3 +1,4 @@
+import { throttle } from 'lodash';
```

Comment:
lodash는 자동 최적화가 안 되는 것 같아서... 이렇게 하는 게 좋을 거예요
```suggestion
import throttle from 'lodash/throttle';
```